EnvelopedEvent is no longer exported with v3.21.2

### `@slack/bolt` version

3.21.2

### Your `App` and Receiver Configuration

```ts
import { type EnvelopedEvent } from '@slack/bolt';

const parseEventJson = (body: string) => {
  const slackEvent: EnvelopedEvent = JSON.parse(body);
};
```

### Node.js runtime version

v20.10.0

### Steps to reproduce:

1. My application works fine with `@slack/bolt` 3.21.1
2. Upgrade `@slack/bolt` to 3.21.2
3. `tsc` throws the following error: `error TS2305: Module '"@slack/bolt"' has no exported member 'EnvelopedEvent'.`

### Expected result:

Backward compatibility should be maintained, and EnvelopedEvent should be importable.

Additionally, EnvelopedEvent was explicitly exported in #1334.

### Actual result:

`EnvelopedEvent` is not exported at 3.21.2.

--- a/src/types/events/index.ts
+++ b/src/types/events/index.ts
@@ -29,7 +29,7 @@ export type FunctionInputs = Record<string, unknown>;
  *
  * This describes the entire JSON-encoded body of a request from Slack's Events API.
  */
-interface EnvelopedEvent<Event = BaseSlackEvent> extends StringIndexed {
+export interface EnvelopedEvent<Event = BaseSlackEvent> extends StringIndexed {
   token: string;
   team_id: string;
   enterprise_id?: string;

